copyright

```diff
@@ -277,6 +277,16 @@ Copyright: Various
 License: public-domain
  Taken from unsplash.com, a collection of photographs that have been donated and
  placed in the public domain.
+ 
+ Files:
+ images/outfit/quarg*
+ images/hardpoint/quarg*
+ images/outfit/small?quarg*
+ images/outfit/medium?quarg*
+ images/outfit/large?quarg*
+Copyright: Evan Fluharty (Evanfluharty@gmail.com)
+License: CC-BY-SA-4.0
+Comment: Derived and completed from works by Maximilian Korber (Wrzlprnft), Fernando S. Sáez (Azure Nyoki), and originally drawn up by Tommy Thach (Bladewood) (all from under the same license)
```

Comment:
I know you operated from preliminary work from the named contributors here when developing the skylance sprite, but is that also true for the rest of the quarg outfits? If not, I imagine you are able to attribute sole copyright to those images.

i.e.
```
Files:
  images/outfit/quarg?skylance
  images/hardpoint/quarg?skylance
Copyright...
License..
Comment: Derived and completed from works by Maximilian Korber (Wrzlprnft), Fernando S. Sáez (Azure Nyoki), and originally drawn up by Tommy Thach (Bladewood) (all from under the same license)

Files:
  images/outfit/quarg*
  ...
Copyright... 
License...
```

Reply:
I'll adjust it more later